crates/solidity/testing/snapshots/cst_output/Statements/recovery_ignore_multiple_empty_matches/generated/0.6.0-failure.yml

```diff
@@ -27,7 +27,7 @@ Tree:
           - (for_keyword꞉ ForKeyword): "for" # (0..3)
           - (open_paren꞉ OpenParen): "(" # (3..4)
           - (initialization꞉ ForStatementInitialization) ► (variant꞉ VariableDeclarationStatement) ► (variable_type꞉ VariableDeclarationType) ► (variant꞉ TypeName) ► (variant꞉ ElementaryType) ► (variant꞉ IntKeyword): "int" # (4..7)
-          - (MISSING): "" # (7..7)
+          - (root꞉ MISSING): "" # (7..7)
```

Comment:
looks like we are missing labels for `MISSING` nodes. I wonder if we are able to use the original label the parser was trying to parse, to indicate what is missing?